cookbook/setup_data/add_translation_packs.rst

```diff
@@ -0,0 +1,25 @@
+
+How to Add Translation Packs
+============================
+
+.. _Crowdin: http://crowdin.net/project/akeneo/
+
+
+Akeneo PIM UI is translated through `Crowdin`_ (feel free to contribute!).
+
+Once a week week, new translation keys are pushed to Crowdin, and new validated translations are pulled to our Github repository.
+
+Akeneo PIM contains translation packs for all languages with more than 80% of translated keys.
+
+When we tag a new minor or patch version, the new translations are available.
```

Comment:
Once a new minor or patch version has been tagged, the new translations are available.